### ISSUE TITLE
Hi, I got a question to ask about the code, thx!

### ISSUE THREAD

**Issue by Ziyu-He** [CalebMigosi/code-more#1]
I am really impressed by your code about the Heston pricing model. But when I am running the code, it says 'WebDriver' object has no attribute 'find_elements_by_xpath'. As a rookie to programming, could you please give me some clue on how to fix that?

**Comment by mzizi-io** [comment]
Hey Ziyu,

lol I must admit I wrote this a long time ago. I can't remember what I was doing here but looking at the error I think this has something to do with the web scrapping part in heston pricing 2. The most likely explanation is that the website I was scrapping from changed their format and so one of the divs does not exist or changed names.

If you are not in a big rush I could check this tonight but if you have some knowledge of html I think you only need to change the name of the class on the div that either changed or disappeared.

My apologies for having not properly documented this. I was also very inexperienced at this time. I do hope that the rest of the material helped in some form or fashion.

**Comment by Ziyu-He** [comment]
> 

Yep! It would be a great help if you could check the code. BTW the rest material helps me a lot. I am really grateful for you to share your resources!
